### DISCUSSION TITLE
Running superset with https

### DISCUSSION THREAD

**Discussion by zeehasham** [apache/superset#17675] [Q&A / Help]
Is there a way to run superset application using https locally? 

Basically I want to embed superset iframe in one on my website and iframe is not allowing http superset link in my https website with iframe.

Any help is appreciated.

**Comment by ziggekatten** [comment]
We run Superset behind Traefik that terminates TLS. This could of course be solved in the same way by nginx or Apache HA-proxy, but regardless, doing this with a reverse proxy is the easiest way.

**Comment by jhult** [comment]
Caddy provides a quick/easy way to get HTTPS going via a reverse proxy: https://caddyserver.com/docs/quick-starts/https#the-reverse-proxy-command

**Comment by nytai** [comment]
While it is possible to handle tls in gunicorn, I highly suggest setting up a reverse proxy like nginx to handle tls certs and terminate SSL there. Nginx or similar technologies are just much better at handling this type of thing than python application servers

**Comment by nytai** [comment]
If you really want to go down the gunicorn route, here's some info https://stackoverflow.com/questions/7406805/running-gunicorn-on-https

**Comment by aukfood** [comment]
Does someone try to set superset behind a reverse proxy like Traefik ? My idea is to access to superset via https url and also to set websocket behind https, all trafik is managed behind Traefik.
Thanks

**Reply by ziggekatten** [reply]
Yep, we run it behind Traefik on docker swarm. Quite simple to set up as long as you keep it to a host rule, as changing base path does not work due to some none relative path exist in the Superset code.

**Reply by nyatitilkesh** [reply]
hey can anyone help in this ?
over a video call may be?
